### PR TITLE
Introduces seconds support for ledger actions

### LINKED ISSUE #2188: Adding/editing timestamp in ledger actions. Need seconds accuracy
## Problem Definition

User needs to be able to also add the seconds for the ledger actions. Also same thing for all other date pickers which act as data input.

## Task

Do it

### PR DESCRIPTION
**Pull request by kelsos**
Closes #2188

### CODE REVIEW

**Review by LefterisJP — COMMENTED** [review]
As discussed in discord the same needs to be done for anything that is adding timestamps to events that go in the DB.

So that also includes the manual trades. And from what I see it does not work there yet.
Any other place where we do that?

Also for other date time pickers, like say the range of a graph this precision is not needed.

File: docs/changelog.rst [hunk 1]
```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :feature:`2188` Users can now specify the exact second a ledger action happened when adding or editing ledger actions.
```
Comment (LefterisJP) [inline docs/changelog.rst:5]: When you also make it work for trades (and any other events) rephrase to:

Users can now specify the datetime for events (ledger actions, trades) to seconds precision.

**Comment by kelsos** [comment]
> Also for other date time pickers, like say the range of a graph this precision is not needed.
Well that is exactly why there is no change in the precision in the graph/profit and loss custom period date picker

**Comment by LefterisJP** [comment]
> > Also for other date time pickers, like say the range of a graph this precision is not needed.
> 
> Well that is exactly why there is no change in the precision in the graph/profit and loss custom period date picker

Obviously I did not check it since the test account had no premium key. This is why I am pointing it out in the review for you to make sure. I trust you enough to not even need to check it.

**Review by LefterisJP — APPROVED** [review]
LGTM

**Pull request merged**